staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -528,27 +527,6 @@ func RelativizeAuthInfoLocalPaths(authInfo *clientcmdapi.AuthInfo) error {
 	return nil
 }
 
-func RelativizeConfigPaths(config *clientcmdapi.Config, base string) error {
```

Comment:
By "users" I mean whoeever imports client-go in their own project, not necessarily in k8s.io/kuberentes.